contributing/documentation/overview.rst

```diff
@@ -258,6 +258,24 @@ link displayed for Platform.sh service.
     Only Pull Requests to maintained branches are automatically built by
     Platform.sh. Check the `roadmap`_ for maintained branches.
 
+Build Documentation Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively you can build the documentation in your own computer following
+these steps:
+
+**Step 1.** Install `pip`_ as explained in the `pip installation`_ article.
+
+**Step 2.** Install `Sphinx`_ and `Sphinx Extensions for PHP and Symfony`_
+(depending on your system, you may need to execute this command as root user):
+
+.. code-block:: bash
+
+    $ pip install sphinx~=1.3.0 git+https://github.com/fabpot/sphinx-php.git
+
+**Step 3.** Run ``make html`` and view the generated documentation in the
```

Comment:
Maybe it should be mentioned that `make html` has to be run inside the `_build' dir.